parser/src/powdr.lalrpop

```diff
@@ -256,13 +261,15 @@ pub Instruction: Instruction<T> = {
 }
 
 pub LinkDeclaration: MachineStatement<T> = {
-    <start:@L> "link" <flag:Expression> "=>" <to:CallableRef> ";" => MachineStatement::LinkDeclaration(ctx.source_ref(start), LinkDeclaration { flag, to })
+    <start:@L> "link" <flag:Expression> "=>" <to:CallableRef> ";" => MachineStatement::LinkDeclaration(ctx.source_ref(start), LinkDeclaration { flag, to, permutation: false, }),
+    <start:@L> "link" <flag:Expression> "~>" <to:CallableRef> ";" => MachineStatement::LinkDeclaration(ctx.source_ref(start), LinkDeclaration { flag, to, permutation: true, }),
```

Comment:
Especially here, I would have thought `<=>` to be more natural.

Reply:
maybe we could actually use `is` and `in` just like in pil?

Reply:
I'm fine with `=>` and `~>` but just as a note: The way instructions are declared was also chosen semi-arbitrarily, and it is easy to change.